backend/src/main/java/ai/verta/modeldb/experimentRun/ExperimentRunDAORdbImpl.java

```diff
@@ -1312,6 +1325,63 @@ public ExperimentRunPaginationDTO findExperimentRuns(FindExperimentRuns queryPar
     }
   }
 
+  private Map<String, List<KeyValue>> getExperimentRunHyperparameterConfigBlobMap(
+      Session session, List<ExperimentRunEntity> experimentRunEntities) {
+
+    String queryBuilder =
+        "Select vme.experimentRunEntity.id, cb From ConfigBlobEntity cb LEFT JOIN VersioningModeldbEntityMapping vme ON vme.blob_hash = cb.blob_hash WHERE cb.hyperparameter_type = :hyperparameterType AND vme.experimentRunEntity.id IN (:expRunIds)";
```

Comment:
this query is getting a lot of unnecessary data because of the left join
the config blogEntity which do not match are also selected.

Reply:
if there is a config blob not associated to an experiment run the code will break

Reply:
fixed